docs/application/native/guides/ui/tizen-ws-shell/tzsh-softkey.md

```diff
@@ -0,0 +1,213 @@
+# TZSH-Softkey
+
+TZSH-Softkey is a library to control the Softkey service window that shows the software buttons such as back button and home button. You can use TZSH-Softkey to control the Softkey service window's visible state, expandable state and opacity state.
+Each state has the following properties:
+
+- **Visible State**: You can show or hide the Softkey service window. If you set the visible state to show, the Softkey service window appears. If you set it to hide, the Softkey service window disappears.
+
+- **Expandable State**: You can control expandable state of the Softkey service window.
+If you set the expand state to on, then the expandable button of the Softkey service window appears as follows:
+![Expand On Transparent](./media/tzsh_softkey_expand_on_transparent.png)
+If you set the expand state to off, then the expandable button disappears as follows:
+![Expand Off Transparent](./media/tzsh_softkey_expand_off_transparent.png)
+
+- **Opacity State**: You can control the opacity state of the Softkey service window.
+If you set the opacity state to opaque, the background of the Softkey service window changes to 24-bit color opaque window as follows:
+![Expand Off Opaque](./media/tzsh_softkey_expand_off_opaque.png)
+If you set the opacity state to transparent, the background changes to 32-bit colored transparent window as follows:
+![Expand Off Transparent](./media/tzsh_softkey_expand_off_transparent.png)
+
+> **Note**
```

Comment:
Please change as:
> **Note**
>
> Each state can present itself differently depending on the implementation of the Softkey service.
> However, the preceding example does not represent all the Softkey service window.

Reply:
Done.

Reply:
Please can you remove extra ">" from line number 24.

Reply:
Removed.

Reply:
@sabeena21 
Isn't the "_Each state can present itself differently depending on the implementation of the Softkey service._" is the reason of "_the preceding example does not represent all the Softkey service window._"?

As "_Each state can present itself differently depending on the implementation of the Softkey service._", it is impossible to show every case of the Softkey service window. It depends on the implementation case of that moment. **So** _the example can't represent all the Softkey service window._

Also, I suggest to move this note to line 5 (before the State list) with removing "preceding "

@surbhig12 I hope you to check this comment, too.

Reply:
@mijin85cho,
If we don't want to use "However" in the sentence.
I agree with your suggestion of removing the word "preceding"  in the sentence and adding word "So"

@juns-kim: Please can you change to:
Each state can present itself differently depending on the implementation of the Softkey service. So the example does not represent all the Softkey service window.